cranelift/module/src/backend.rs

```diff
@@ -179,6 +179,10 @@ where
 pub fn default_libcall_names() -> Box<dyn Fn(ir::LibCall) -> String> {
     Box::new(move |libcall| match libcall {
         ir::LibCall::Probestack => "__cranelift_probestack".to_owned(),
+        ir::LibCall::UdivI64 => "udivdi3".to_owned(),
+        ir::LibCall::SdivI64 => "divdi3".to_owned(),
+        ir::LibCall::UremI64 => "umoddi3".to_owned(),
+        ir::LibCall::SremI64 => "moddi3".to_owned(),
```

Comment:
```suggestion
        ir::LibCall::UdivI64 => "__udivdi3".to_owned(),
        ir::LibCall::SdivI64 => "__divdi3".to_owned(),
        ir::LibCall::UremI64 => "__umoddi3".to_owned(),
        ir::LibCall::SremI64 => "__moddi3".to_owned(),
```

Reply:
Oh sorry, I completely forgot that `ceil` isn't a builtin and thought the underscores are added elsewhere. Fixed.